src/quadspline.rs

```diff
@@ -0,0 +1,20 @@
+//! Quadratic Bézier splines.
+use crate::Point;
+
+#[derive(Clone, Debug, PartialEq)]
+#[allow(missing_docs)]
+pub struct QuadSpline(Vec<Point>);
+
+impl QuadSpline {
+    /// Construct a new quadratic spline from an array of points.
```

Comment:
```suggestion
    /// Construct a new `QuadSpline` from an array of [`Point`]s.
```

might as well link this to the docs for `Point`